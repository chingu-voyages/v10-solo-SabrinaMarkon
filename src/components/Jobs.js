import React, { Component } from 'react';

export default class Jobs extends Component {
  constructor(props) {
    super(props);


  }

  render() {
    return (
      <div>
        <h1>Jobs</h1>
      </div>
    )
  }
}