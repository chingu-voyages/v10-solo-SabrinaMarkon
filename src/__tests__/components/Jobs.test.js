import React from 'react';
import { shallow } from 'enzyme';

// components
import Jobs from '../components/Jobs';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Jobs />);
});

describe('Testing Jobs component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('Jobs'));
  });
});