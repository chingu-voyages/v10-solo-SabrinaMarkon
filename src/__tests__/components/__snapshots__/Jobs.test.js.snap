// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Jobs component it should match the Jobs snapshot 1`] = `undefined`;
